integration_test/generate_expected_metrics/generate_expected_metrics.go

```diff
@@ -0,0 +1,274 @@
+//go:build integration_test
+
+/*
+PROJECT: What GCP project to use.
+GOOGLE_APPLICATION_CREDENTIALS: Path to a credentials file for interacting with
+    some GCP services. All gcloud commands actually use a different set of
+    credentials, those in CLOUDSDK_CONFIG (unfortunately).
```

Comment:
I guess remove the comment about "all gcloud commands", since this tool doesn't use gcloud.